# -*- coding: utf-8 -*-


from PySide2 import QtCore, QtGui, QtWidgets

class Ui_MainWindow(object):
    def setupUi(self, MainWindow):
        MainWindow.setObjectName("MainWindow")
        MainWindow.resize(982, 522)
        self.centralwidget = QtWidgets.QWidget(MainWindow)
        self.centralwidget.setObjectName("centralwidget")
        self.frame = QtWidgets.QFrame(self.centralwidget)
        self.frame.setGeometry(QtCore.QRect(20, 60, 261, 381))
        self.frame.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.frame.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame.setObjectName("frame")
        self.label = QtWidgets.QLabel(self.frame)
        self.label.setGeometry(QtCore.QRect(10, 10, 81, 41))
        self.label.setObjectName("label")
        self.comboBox_path_list = QtWidgets.QComboBox(self.frame)
        self.comboBox_path_list.setGeometry(QtCore.QRect(100, 10, 131, 41))
        self.comboBox_path_list.setObjectName("comboBox_path_list")
        self.comboBox_path_list.addItem("")
        self.comboBox_path_list.addItem("")
        self.label_2 = QtWidgets.QLabel(self.frame)
        self.label_2.setGeometry(QtCore.QRect(21, 69, 61, 41))
        self.label_2.setObjectName("label_2")
        self.label_3 = QtWidgets.QLabel(self.frame)
        self.label_3.setGeometry(QtCore.QRect(20, 130, 71, 51))
        self.label_3.setObjectName("label_3")
        self.label_5 = QtWidgets.QLabel(self.frame)
        self.label_5.setGeometry(QtCore.QRect(20, 190, 71, 41))
        self.label_5.setObjectName("label_5")
        self.lineEdit_path = QtWidgets.QLineEdit(self.frame)
        self.lineEdit_path.setGeometry(QtCore.QRect(100, 70, 131, 41))
        self.lineEdit_path.setObjectName("lineEdit_path")
        self.lineEdit_start = QtWidgets.QLineEdit(self.frame)
        self.lineEdit_start.setGeometry(QtCore.QRect(100, 130, 131, 41))
        self.lineEdit_start.setObjectName("lineEdit_start")
        self.lineEdit_step = QtWidgets.QLineEdit(self.frame)
        self.lineEdit_step.setGeometry(QtCore.QRect(100, 190, 131, 41))
        self.lineEdit_step.setObjectName("lineEdit_step")
        self.pushButton_start_game = QtWidgets.QPushButton(self.frame)
        self.pushButton_start_game.setGeometry(QtCore.QRect(20, 260, 211, 41))
        self.pushButton_start_game.setObjectName("pushButton_start_game")
        self.pushButton_clear = QtWidgets.QPushButton(self.frame)
        self.pushButton_clear.setGeometry(QtCore.QRect(20, 320, 211, 41))
        self.pushButton_clear.setObjectName("pushButton_clear")
        self.label_4 = QtWidgets.QLabel(self.centralwidget)
        self.label_4.setGeometry(QtCore.QRect(430, 10, 201, 41))
        self.label_4.setObjectName("label_4")
        self.label_6 = QtWidgets.QLabel(self.centralwidget)
        self.label_6.setGeometry(QtCore.QRect(100, 20, 111, 31))
        self.label_6.setObjectName("label_6")
        self.label_7 = QtWidgets.QLabel(self.centralwidget)
        self.label_7.setGeometry(QtCore.QRect(780, 10, 181, 41))
        self.label_7.setObjectName("label_7")
        self.plainTextEdit_original_people = QtWidgets.QPlainTextEdit(self.centralwidget)
        self.plainTextEdit_original_people.setGeometry(QtCore.QRect(350, 60, 251, 351))
        self.plainTextEdit_original_people.setPlainText("")
        self.plainTextEdit_original_people.setObjectName("plainTextEdit_original_people")
        self.plainTextEdit_out_people = QtWidgets.QPlainTextEdit(self.centralwidget)
        self.plainTextEdit_out_people.setGeometry(QtCore.QRect(680, 60, 241, 351))
        self.plainTextEdit_out_people.setPlainText("")
        self.plainTextEdit_out_people.setObjectName("plainTextEdit_out_people")
        MainWindow.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(MainWindow)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 982, 18))
        self.menubar.setObjectName("menubar")
        MainWindow.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(MainWindow)
        self.statusbar.setObjectName("statusbar")
        MainWindow.setStatusBar(self.statusbar)

        self.retranslateUi(MainWindow)
        QtCore.QMetaObject.connectSlotsByName(MainWindow)

    def retranslateUi(self, MainWindow):
        MainWindow.setWindowTitle(QtWidgets.QApplication.translate("MainWindow", "MainWindow", None, -1))
        self.label.setText(QtWidgets.QApplication.translate("MainWindow", "path_list", None, -1))
        self.comboBox_path_list.setItemText(0, QtWidgets.QApplication.translate("MainWindow", "person.txt", None, -1))
        self.comboBox_path_list.setItemText(1, QtWidgets.QApplication.translate("MainWindow", "person.csv", None, -1))
        self.label_2.setText(QtWidgets.QApplication.translate("MainWindow", "path", None, -1))
        self.label_3.setText(QtWidgets.QApplication.translate("MainWindow", "start", None, -1))
        self.label_5.setText(QtWidgets.QApplication.translate("MainWindow", "step", None, -1))
        self.lineEdit_path.setText(QtWidgets.QApplication.translate("MainWindow", "person.txt", None, -1))
        self.lineEdit_start.setText(QtWidgets.QApplication.translate("MainWindow", "1", None, -1))
        self.lineEdit_step.setText(QtWidgets.QApplication.translate("MainWindow", "2", None, -1))
        self.pushButton_start_game.setText(QtWidgets.QApplication.translate("MainWindow", "start_game", None, -1))
        self.pushButton_clear.setText(QtWidgets.QApplication.translate("MainWindow", "clear", None, -1))
        self.label_4.setText(QtWidgets.QApplication.translate("MainWindow", "original_people", None, -1))
        self.label_6.setText(QtWidgets.QApplication.translate("MainWindow", "init", None, -1))
        self.label_7.setText(QtWidgets.QApplication.translate("MainWindow", "out_people", None, -1))

